catalog/spatial/registry/registry-federation-admin-impl/src/main/java/org/codice/ddf/registry/federationadmin/impl/FederationAdmin.java

```diff
@@ -411,6 +414,15 @@ public boolean registryStatus(String servicePID) {
         return nodes;
     }
 
+    @Override
+    public void regenerateRegistrySources() {
+        try {
+            sourceConfigRefresh.regenerateAllSources();
+        } catch (FederationAdminException e){
+            LOGGER.error("Error regenerating registry sources.", e);
```

Comment:
You may want to revisit the logging guidance. Might be suitable for a different level if an admin cannot address: https://codice.atlassian.net/wiki/display/DDF/Log+Levels#LogLevels-Error